docs/devguide/internet_enabled_rtps.rst

```diff
@@ -119,7 +119,8 @@ As an example:
 
 Each participant should use a single RtpsRelay instance due to the way NAT bindings work.
 Most firewalls will only forward packets received from the destination address that was originally used to create the NAT binding.
-That is, if participant A is interacting with relay A and participant B is interacting with relay B, then a message from A to B must go from A to relay A, to relay B, and finally to B.  Relay A cannot send directly to B since that packet will not be accepted by the firewall.
+That is, if participant A is interacting with relay A and participant B is interacting with relay B, then a message from A to B must go from A to relay A, to relay B, and finally to B.
+Relay A cannot send directly to B since that packet will not be accepted by the firewall.
```

Comment:
```suggestion
That is, if participant A is interacting with relay 1 and participant B is interacting with relay 2, then a message from A to B must go from A to relay 1, to relay 2, and finally to B.
Relay 1 cannot send directly to B since that packet will not be accepted by the firewall.
```